File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    ans = [0, 0]
    for i in sizes:
        i.sort()

    for i in sizes:
        ans[0] = max(i[0], ans[0])
        ans[1] = max(i[1], ans[1])
        
    return ans[0] * ans[1]

File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    sizes.forEach((item) => {
        item.sort((a, b) => a - b);
    });
    
    const maxSize = [0, 0];
    
    sizes.forEach(([x, y]) => {
        maxSize[0] = Math.max(maxSize[0], y);
        maxSize[1] = Math.max(maxSize[1], x);
    })
    
    return maxSize[0] * maxSize[1];
}